example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -144,6 +144,7 @@
 				3B06AD1E1E4923F5004D2608 /* Thin Binary */,
 				17130ABE28ACDC4A007FE54B /* Square setup script */,
 				324E410A8DA310037B0B821B /* [CP] Embed Pods Frameworks */,
+				CB238059B6C64501CA7FAC41 /* [CP] Copy Pods Resources */,
```

Comment:
I am not totally sure we want this at the end (does the `setup` script fail?)` Need to check with @brandonjenniges on his thoughts.

Reply:
Thanks for confirming!